sdmetrics/reports/multi_table/_properties/base.py

```diff
@@ -11,13 +11,66 @@ class BaseMultiTableProperty():
     Attributes:
         properties (dict):
             A dict mapping the table names to their single table properties.
+        is_computed (bool):
+            Whether or not the property has been computed.
+        _only_multi_table (bool):
+            Whether or not the property only exists for multi-tables.
+        details (pandas.DataFrame):
+            The multi table details property dataframe.
     """
 
     _single_table_property = None
+    _num_iteration_case = None
 
     def __init__(self):
         self._properties = {}
         self.is_computed = False
+        self._only_multi_table = False
```

Comment:
If I'm not mistaken, I think you can use if the `self._num_iteration_case == 'relationship'` and get rid of this. My assumption is that if we're looping at relationships, there can't be a single table version